### DISCUSSION TITLE
Plugin to browse services such as Unsplash.

### DISCUSSION THREAD

**Discussion by DennisSuitters** [Studio-42/elFinder#3314] [Ideas]
Hi @nao-pon 
Is it possible to create plugins, such as an ability to browse online services such as Unsplash to link to, or download to a server within elFinder. I was thinking of having something appear in the folders area on the left "Unsplash", or whatever service, that when selected could then fetch and display images in the file pane with an option to download to server, or link directly, and possibly allow the search function built into elFInder to interact with fetched images.
Something like the "Free Images" Tab, here https://scaleflex.github.io/filerobot-uploader/

**Comment by nao-pon** [comment]
@DiemenDesign For now, I have no idea, but #3022 is relevant.
